app/business/rankings.py

```diff
@@ -227,13 +228,19 @@ def __init__(self, **kwargs):
         self.numerical_rank   = '-1'
 
 
-def _build_PersonalBestRecord(query_tuple):
+def _build_PersonalBestRecord(record):
     """ Builds a PersonalBestRecord from the 5-tuple returned from the ordered PB queries below.
     The tuple looks like (user_id, single/average, comp_id, comp_title, username). """
 
-    user_id, result, comp_id, comp_title, username, comment, user_is_verified = query_tuple
-    return PersonalBestRecord(personal_best=result, user_id=user_id, username=username, comp_id=comp_id,
-        comp_title=comp_title, comment=comment, user_is_verified=user_is_verified)
+    return PersonalBestRecord(
+        personal_best=record.single,
+        user_id=record.user_id,
+        username=record.username,
```

Comment:
This is where it's dying when I try to spin up. The record you're passing in doesn't have a `username` attribute.